core/templates/components/forms/schema-based-editors/schema-based-list-editor.directive.ts

```diff
@@ -81,6 +84,34 @@ angular.module('oppia').directive('schemaBasedListEditor', [
           return false;
         };
 
+        var initializeoldToNewAnswerChoicesMapping = function() {
+          if ($scope.oldToNewAnswerChoicesMapping !== undefined) {
+            for (var i = 0; i < $scope.localValue.length; i++) {
+              $scope.oldToNewAnswerChoicesMapping.
+                newToOldAnswerChoicesPostion.push(i);
+            }
+          }
+        };
+        var updateoldToNewAnswerChoicesMappingOnDeleteElement =
+            function(index) {
+              if ($scope.oldToNewAnswerChoicesMapping !== undefined) {
+                var deletedIndex = $scope.oldToNewAnswerChoicesMapping.
+                  newToOldAnswerChoicesPostion[index];
+                if (deletedIndex !== -1) {
+                  $scope.oldToNewAnswerChoicesMapping.deletedIndexes.push(
+                    deletedIndex);
+                }
+                $scope.oldToNewAnswerChoicesMapping.
+                  newToOldAnswerChoicesPostion.splice(index, 1);
+              }
+            };
+        var updateoldToNewAnswerChoicesMappingOnAddElement = function() {
```

Comment:
updateAnswerChoicesMappingOnAddItem for parity.


Also, "old" is not capitalized in the existing name.

Reply:
done